radio/src/gui/480x272/widget.h

```diff
@@ -45,7 +45,7 @@ class Widget
     {
     }
 
-    virtual void update() const
+    virtual void update()
```

Comment:
This is good change. I had to remove const here because Lua widget will modify itself on update() if it encounters any error. I guess my compiler did not catch this.